### ISSUE TITLE
RDP + Local User this PC win10

### ISSUE THREAD

**Issue by edgiru** [stascorp/rdpwrap#767]
you can not work a local user on the computer and the RDP user simultaneously. If the RDP user enters the computer over the network under the name for example: "buh" of the local user for example: "Administrator" switches to "buh". The local user and the RDP user see the same desktop.

**Comment by lehha** [comment]
I have a similar problem: if you work locally, and then connect via RDP, then a new session starts with this user (in parallel, the current one does not throw out). This with option "Single session per user"

**Comment by johnnyontheweb** [comment]
Same problem for me. I disabled multiple access to same user but nothing. I'm running Windows 10 home 1903 with INI file.
it's really annoying, Onedrive resets every time a multiple session for same user has started.
![Cattura](https://user-images.githubusercontent.com/10365055/59877142-7bf43680-93a5-11e9-86d0-a10221d8105c.PNG)

can you help?

**Comment by Mayar** [comment]
Нas anyone found the solution?

**Comment by edgiru** [comment]
> Нas anyone found the solution?

no solution yet

**Comment by Mayar** [comment]
How to workaround the problem? Change .dll?

**Comment by s0nic9** [comment]
no workaround, rdpwrap works with updates from others users, read issues carefully.

**Comment by Mayar** [comment]
На всех машинах такая байда. Как решали, коллеги?

**Comment by sebaxakerhtc** [comment]
Duplicate of https://github.com/stascorp/rdpwrap/issues/720
